### PR TITLE
DRAFT Compiler FE: Support Less equal

### PR DESCRIPTION
**Pull request by ko-dimi**
Add support for Less Equal operator. 

ONE-DCO-1.0-Signed-off-by: Dmitry Kozlov <kozlov.dm@partner.samsung.com>

### CODE REVIEW

**Comment by ko-dimi** [comment]
@nnfw-bot test signed-off-by-checker

File: compiler/luci/import/src/Nodes/CircleLessEqual.cpp [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  return tensors[outputs[0]]->type == circle::TensorType::TensorType_BOOL;
+}
+
+CircleNode *CircleLessEqualGraphBuilder::build_node(const circle::OperatorT &,
+                                                    const std::vector<CircleNode *> &inputs,
+                                                    loco::Graph *graph) const
+{
+  auto *node = graph->nodes()->create<CircleEqual>();
```
Comment (d-krylov) [inline compiler/luci/import/src/Nodes/CircleLessEqual.cpp:55]:
```suggestion
  auto *node = graph->nodes()->create<CircleLessEqual>();
```

File: compiler/tflchef/tflite/src/Op/LessEqual.h [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#ifndef __TFLITE_OP_LESSEQUAL_H__
+#define __TFLITE_OP_LESSEQUAL_H__
+
+#include "TFliteOpChef.h"
+
+namespace tflchef
+{
+
+/**
+ * @brief tflchef operator builder for Greater Equal
```
Comment (toomuchsalt) [inline compiler/tflchef/tflite/src/Op/LessEqual.h:26]:
```suggestion
 * @brief tflchef operator builder for LessEqual Equal
```

File: compiler/tflchef/proto/tflchef.proto [hunk 3]
```diff
@@ -462,6 +466,7 @@ message Operation {
   // ScatterNdOptions 200
   optional NotEqualOptions notequal_options = 201;
   optional ExpandDimsOptions expand_dims_options = 202;
+  // LessEqualOptions 203
```
Comment (toomuchsalt) [inline compiler/tflchef/proto/tflchef.proto:469]:
```suggestion
```

File: compiler/luci/import/src/GraphBuilderRegistry.cpp [hunk 4]
```diff
@@ -27,35 +27,36 @@ GraphBuilderRegistry::GraphBuilderRegistry()
...
+  CIRCLE_NODE(FLOOR_MOD, CircleFloorModGraphBuilder);                // 95
+  CIRCLE_NODE(FULLY_CONNECTED, CircleFullyConnectedGraphBuilder);    // 9
+  CIRCLE_NODE(GATHER, CircleGatherGraphBuilder);                     // 36
+  CIRCLE_NODE(GATHER_ND, CircleGatherNdGraphBuilder);                // 107
+  CIRCLE_NODE(GREATER, CircleGreaterGraphBuilder);                   // 61
+  CIRCLE_NODE(GREATER_EQUAL, CircleGreaterEqualGraphBuilder);        // 62
+  CIRCLE_NODE(IF, CircleIfGraphBuilder);                             // 118
+  CIRCLE_NODE(LEAKY_RELU, CircleLeakyReluGraphBuilder);              // 98,
+  CIRCLE_NODE(LESS, CircleLessGraphBuilder);                         // 58
+  CIRCLE_NODE(LESS_EQUAL, CircleLessEqualGraphBuilder);
```
Comment (seanshpark) [inline compiler/luci/import/src/GraphBuilderRegistry.cpp:59]: please add "// number"

File: compiler/luci/import/src/GraphBuilderRegistry.cpp [hunk 5]
```diff
@@ -110,13 +111,18 @@ GraphBuilderRegistry::GraphBuilderRegistry()
 
   // BuiltinOperator_DEQUANTIZE = 6,
   // BuiltinOperator_EMBEDDING_LOOKUP = 7,
+  // BuiltinOperator_FLOOR = 8,
```
Comment (seanshpark) [inline compiler/luci/import/src/GraphBuilderRegistry.cpp:114]: Don't add these comments

File: compiler/luci/import/src/Nodes/CircleLessEqual.cpp [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  const auto &tensors = args.reader.tensors();
+
+  if (tensors[inputs[0]]->type != tensors[inputs[1]]->type)
+  {
+    return false;
+  }
+
+  if (outputs.size() != 1)
```
Comment (seanshpark) [inline compiler/luci/import/src/Nodes/CircleLessEqual.cpp:43]: put this check after input checkf like the others

File: compiler/luci/lang/include/luci/IR/Nodes/CircleLessEqual.h [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+#include "luci/IR/CircleNodeDecl.h"
+#include "luci/IR/CircleOpcode.h"
+
+#include "luci/IR/LuciNodeMixins.h"
+
+namespace luci
+{
+
+/**
+ * @brief LESS EQUAL in Circle
```
Comment (seanshpark) [inline compiler/luci/lang/include/luci/IR/Nodes/CircleLessEqual.h:29]: `LESS_EQUAL`

**Comment by kvochko** [comment]
@seanshpark I will be taking over this task, and unfortunately I don't have write access to the source branch of this PR, so closing this one in favor of #1976

**Pull request closed without merging**